test/unit/__snapshots__/switch.spec.ts.snap

```diff
@@ -1,52 +1,25 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`switch not allowed in 5.1: code 1`] = `
-"local ____exports = {}
-function ____exports.__main(self)
-    local ____switch3 = \\"abc\\"
-    goto ____switch3_end
-    ::____switch3_end::
-end
-return ____exports"
-`;
-
-exports[`switch not allowed in 5.1: diagnostics 1`] = `"main.ts(2,9): error TSTL: Switch statements is/are not supported for target Lua 5.1."`;
-
 exports[`switch uses elseif 1`] = `
 "local ____exports = {}
 function ____exports.__main(self)
     local result = -1
-    local ____switch3 = 2
-    if ____switch3 == 0 then
-        goto ____switch3_case_0
-    elseif ____switch3 == 1 then
-        goto ____switch3_case_1
-    elseif ____switch3 == 2 then
-        goto ____switch3_case_2
-    end
-    goto ____switch3_end
-    ::____switch3_case_0::
-    do
-        do
-            result = 200
-            goto ____switch3_end
-        end
-    end
-    ::____switch3_case_1::
-    do
-        do
-            result = 100
-            goto ____switch3_end
-        end
-    end
-    ::____switch3_case_2::
     do
-        do
-            result = 1
-            goto ____switch3_end
+        local ____switch3 = 2
+        if ____switch3 == 0 then
+            do
```

Comment:
Do we still need these `do` blocks now that we encapsulate everything in if scopes already?

Reply:
These do statements are here because the test defines the cases like so...
```ts
case 1: {
  result = 100
  break;
}
```

The parens create the do in the above case, the below would be sans do block.
```ts
case 1:
  result = 100
  break;
```

Reply:
I went ahead and removed this snapshot btw, created a better one to illustrate the many cases we had talked through already.